test/parallel/test-dgram-empty-packet.js

```diff
@@ -1,22 +1,22 @@
 'use strict';
-var common = require('../common');
+const common = require('../common');
+const dgram = require('dgram');
 
-var dgram = require('dgram');
-var callbacks = 0;
-var client;
-var timer;
+let callbacks = 0;
+let timer;
 
 if (common.isOSX) {
   common.skip('because of 17894467 Apple bug');
   return;
 }
 
-client = dgram.createSocket('udp4');
+const client = dgram.createSocket('udp4');
+
+client.bind(0, function() { 
```

Comment:
@Trott sorry for that!
git commit --amend
make lint
./configure && make -j8 test

Total errors found: 0
git push --force-with-lease origin dgram-empty-es6

Tested and pushed!